### PR TITLE
updated gateway for merra data

### PR DESCRIPTION
**Pull request by aishwaryavijaysinhasane**

### CODE REVIEW

File: gateway-service/routes/data_retrieval.py [hunk 1]
```diff
@@ -28,20 +28,37 @@ def createWeatherData():
...
+                "station":request_body.get('station')
+            }, default=json_util.default).encode('utf-8'))
+            
+            kafka_producer.send('audit-queue', json.dumps({
+                "user_id": user_id,
+                "response": {},
+                "request": request.data,
+                "service_provider_identifier": 'data-retrieval-service'
+            }, default=json_util.default).encode('utf-8'))   
+        else:
```
Comment (impiyush83) [inline gateway-service/routes/data_retrieval.py:49]: not else should be elif type == 'merra'

File: gateway-service/routes/data_retrieval.py [hunk 2]
```diff
@@ -28,20 +28,37 @@ def createWeatherData():
     try:
         session_service_response = check_user_session(request.headers.get('Access-Token'))
         request_body = json.loads(request.data)
+        type = request_body.get('type')
```
Comment (impiyush83) [inline gateway-service/routes/data_retrieval.py]: no need of variable can be directly accessed by request_body.get('type')

File: gateway-service/routes/data_retrieval.py (hunk 1 above)
Comment (impiyush83) [inline gateway-service/routes/data_retrieval.py]: should be elif type == 'merra
'

File: gateway-service/routes/data_retrieval.py [hunk 3]
```diff
@@ -28,20 +28,37 @@ def createWeatherData():
...
-        }, default=json_util.default).encode('utf-8'))
+        if type == 'nexrad':
         
-        kafka_producer.send('audit-queue', json.dumps({
-            "user_id": user_id,
-            "response": {},
-            "request": request.data,
-            "service_provider_identifier": 'data-retrieval-service'
-        }, default=json_util.default).encode('utf-8'))   
+            kafka_producer.send('data-retrieval-queue', json.dumps({
```
Comment (impiyush83) [inline gateway-service/routes/data_retrieval.py]: just assign JSON inside to variable
example :

if type == 'nexrad':
     j1 = json.dumps({"acv": 'x1'})
elif type == 'merra':
   j1 = json.dumps({"wfwe": "few"})

kafka_producer.send('queue_name', j1)
kafka_producer.send('queue_namne2', j2)

**Review by impiyush83 — CHANGES_REQUESTED** [review]

**Review by impiyush83 — APPROVED** [review]

**Review by RushikeshPharate — APPROVED** [review]

**Pull request merged**